masterlist.yaml

```diff
@@ -24664,3 +24973,276 @@ plugins:
   - name: 'MoonAndStar_MAS.esp'
     after:
       - 'AuroraVillage.esp'
+  - name: 'No Vendor or Loot Spells.esp'
+    global_priority: 105
+  - name: 'Skyrim Unlocked.esp'
+    global_priority: 70
+    inc:
+      - name: "Open Cities Skyrim.esp"
+        condition: 'not active("Open Cities Skyrim - Skyrim Unlocked Patch.esp")'
+    msg:
+      - type: warn
+        condition: 'active("Open Cities Skyrim.esp") and not active("Open Cities Skyrim - Skyrim Unlocked Patch.esp")'
+        content: 'Incompatible with: Open Cities Skyrim. You must install the compatibility patch.'
+      - <<: *reqRequiemPatch
+        condition: 'file("Requiem.esp") and not active("Requiem - Skyrim Unlocked.esp") and not active("Requiem - Skyrim Unlocked - OCS.esp")'
+  - name: '83Willows_101BUGS_V4_HighRes_HighSpawn.esp' 
+    msg:
+      - <<: *reqRequiemPatch
+        condition: 'file("Requiem.esp") and not active("Requiem - 83Willows 101Bugs.esp")'
+      - <<: *redundantPluginX
```

Comment:
`Requiem - 83Willows 101Bugs.esp` is both a patch and a replacer plugin (but with a different name than the original plugin). The rule is:
Do not use `83Willows_101BUGS_V4_HighRes_HighSpawn.esp` if `Requiem - 83Willows 101Bugs.esp` is installed.
and not:
Use only one of `Requiem - 83Willows 101Bugs.esp` and `83Willows_101BUGS_V4_HighRes_HighSpawn.esp`.